terraform/helm/aptos-node/values.yaml

```diff
@@ -41,8 +41,13 @@ haproxy:
   affinity: {}
   limits:
     validator:
```

Comment:
If these values make for a sane default, can we just hardcode them in the actual HAProxy config, rather than template them in via Helm? Makes the config more readable and easier to diff with the docker-compose HAProxy config

Reply:
That is something to consider. But I would do this in a separate PR.